src/GafferImage/OpenImageIOReader.cpp

```diff
@@ -553,15 +566,32 @@ void OpenImageIOReader::hashDataWindow( const GafferImage::ImagePlug *output, co
 	hashFileName( context, h );
 	refreshCountPlug()->hash( h );
 	missingFrameModePlug()->hash( h );
+
+	std::string fileName = fileNamePlug()->getValue();
+	MissingFrameMode mode = (MissingFrameMode)missingFrameModePlug()->getValue();
```

Comment:
Because you're getting the value from `missingFrameModePlug()` here, there's no need to also do `missingFrameModePlug()->hash()` previously.